man/man5/zpool-features.5

```diff
@@ -589,5 +591,51 @@ files.
 
 .RE
 
+.sp
+.ne 2
+.na
+\fB\fBencryption\fR\fR
+.ad
+.RS 4n
+.TS
+l l .
+GUID	com.datto:encryption
+READ\-ONLY COMPATIBLE	no
+DEPENDENCIES	extensible_dataset
+.TE
+
+This feature enables the creation of natively encrypted datasets. ZFS will
+encrypt all user data including file and zvol data, file attributes, ACLs,
+permission bits, directory listings, and FUID mappings. ZFS will not encrypt
+metadata related to the pool structure, including dataset names, dataset
+hierarchy, file size, file holes, and dedup tables. Key rotation is managed
+internally by the ZFS kernel module and changing the user's key does not
+require re-encrypting the entire dataset. Datasets can be scrubbed, resilvered,
+moved, and deleted without the encryption keys being loaded (see the zfs key
+subcommand for more info).
```

Comment:
I think move it, and leave just the minimal description of what this property does (allows setting encryption), and when it becomes active and enabled.